src/config.cr

```diff
@@ -1,12 +1,25 @@
 require "./ext/scrypt_password"
 require "json"
 require "kemal_jwt_auth"
+require "dppm/prefix"
 
 struct DppmRestApi::Config
   include JSON::Serializable
   include KemalJWTAuth::UsersCollection
   property groups : Array(Group)
   property users : Array(User)
+  setter prefix : String = DPPM.default_prefix
+  @[JSON::Field(ignore: true)]
+  @_prefix : DPPM::Prefix? = nil
+
+  def prefix : DPPM::Prefix
+    @_prefix ||= DPPM::Prefix.new(@prefix).tap &.create
+  end
+
+  def prefix=(prefix : DPPM::Prefix)
+    @prefix = prefix.path.to_s
+    @_prefix = prefix
+  end
```

Comment:
All of this isn't needed. Just `class_property : DPPM::Prefix { raise "no prefix set" }`.
The prefix don't have to be in the configuration, the sane default prefix managed should be where the `dppm` application is.